assets/src/stories-editor/helpers/index.js

```diff
@@ -1452,3 +1452,20 @@ export const isVideoSizeExcessive = ( media ) => {
 
 	return media.media_details.filesize > media.media_details.length * VIDEO_ALLOWED_MEGABYTES_PER_SECOND * MEGABYTE_IN_BYTES;
 };
+
+/**
+ * Returns a unique ID that is guaranteed to not start with a number.
+ *
+ * Useful for using in HTML attributes.
+ *
+ * @return {string} Unique ID.
+ */
+export const getUniqueId = () => {
+	let uniqueId = uuid();
+
+	while ( /^\d/.test( uniqueId ) ) {
```

Comment:
I suggest a performance improvement here. Instead of a `while` to generate `uuid` until it starts with a letter, do this instead:

```js
return uuid().replace( /^\d/, 'a' );
```

Reply:
Done in d63850e9347767269b8d44f062a0c06884392bae